exporter/awsemfexporter/metric_translator.go

```diff
@@ -331,15 +596,11 @@ func buildCWMetric(dp DataPoint, pmd *pdata.Metric, namespace string, metricSlic
 		// since calculateRate() needs metric name as one of metric identifiers
 		fields[pmd.Name()] = int64(FakeMetricValue)
 		metricVal = metric.Value()
-		if needsCalculateRate(pmd) {
```

Comment:
Is this a breaking change?

Reply:
Bryan don't need to touch this function.